From 34a51590018bd25e99b3452ed2043274231708c6 Mon Sep 17 00:00:00 2001
From: Joelwfl <joel.w.fl@gmail.com>
Date: Mon, 22 Sep 2014 09:26:29 +0200
Subject: [PATCH 2/3] Changed .Xresources.

---
 .Xresources | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.Xresources b/.Xresources
index 805f5f0..0cf0963 100755
--- a/.Xresources
+++ b/.Xresources
@@ -95,6 +95,6 @@ Exscreensaver.passwd.thermometer.width:       8
 !white
 *color7: #eae8e6
 *color15: #cfcdcc
-
+!
 
 
-- 
1.9.1

